exercises/practice/reverse-string/.meta/template-data.yaml

```diff
@@ -0,0 +1,27 @@
+properties:
+  reverse:
+    test: |-
+      sprintf(q:to/END/, %case<description>.raku, %case<input><value>.raku, %case<expected>.raku, %case<input><value>.raku, %case<input><value>.raku);
+      subtest %s => {
+        cmp-ok(
+            switch-round(%s),
+            "eq",
+            %s,
```

Comment:
I think it would be worth adding a simple description to each of these, maybe something like "reverse once" and "reverse twice".